docs/server/source/simple-network-setup.md

```diff
@@ -0,0 +1,197 @@
+# How to setup a BigchainDB Network
+The process to create a network is both *social* and *technical*: *social* because someone (that we will call **Coordinator**) needs to find at least three other **Members** willing to join the network, and coordinate the effort; *technical* because each member of the network needs to set up a machine running BigchainDB. (Note: a Coordinator is a Member as well.)
+
+A **BigchainDB Network** (*Network* henceforth) is a set of **4 or more BigchainDB Nodes** (*Node* henceforth). Every Node is independently managed by a Member, and runs an instance of the [BigchainDB Server software][bdb:software]. At the **Genesis** of a Network, there **MUST** be at least **4** Nodes ready to connect. After the Genesis, a Network can dynamically add new Nodes or remove old Nodes.
+
+A Network will stop working if more than one third of the Nodes are down. The bigger a Network, the more failures it can handle. A Network of size 4 can tolerate only 1 failure, so if 3 out of 4 Nodes are online, everything will work as expected. Eventually, the Node that was offline will automatically sync with the others.
+
+
+## Before we start
+This tutorial assumes you have basic knowledge on how to manage a GNU/Linux machine. The commands are tailored for an up-to-date *Debian-like* distribution (we use an **Ubuntu 17.10** Virtual Machine on Microsoft Azure): if you are on a different distribution you might need to adapt the name of the packages installed.
```

Comment:
I would feel better about suggesting a long-term-support (LTS) version of Ubuntu. Ubuntu 18.04 LTS is out now.

Reply:
I can update to 18.04 LTS

Reply:
Everything works well with Ubuntu 18.04 :1st_place_medal:

Reply:
It still says **Ubuntu 17.10**.